cli/packages/cmd/login.go

```diff
@@ -326,7 +347,8 @@ func askForDomain() error {
 
 	if selectedHostingOption == INFISICAL_CLOUD {
 		//cloud option
-		config.INFISICAL_URL = util.INFISICAL_DEFAULT_API_URL
+		config.INFISICAL_URL = fmt.Sprintf("%s/api", util.INFISICAL_DEFAULT_URL)
```

Comment:
Is there a reason we need to define a new `INFISICAL_DEFAULT_URL` varable? I think we can just re use `INFISICAL_DEFAULT_API_URL`?

Reply:
Oh i see, you need it for the login url

Reply:
Yep